src/api/SystemAPI.ts

```diff
@@ -21,7 +21,7 @@ import Web3 from 'web3';
 
```

Comment:
Can you do me a favor and delete the `testSets.ts` file? Seems like its not used anywhere